deployments/gpu_plugin/overlays/fake_devices/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
```suggestion
kind: Kustomization
nameSuffix: -fake
```

Reply:
@mythi I'm not really sure about adding -fake suffix to all objects: service, serviceAccount, clusterRole, clusterRoleBinding, configMap, daemonSet.

I think it would be rare to run both fake and real GPU plugin at the same time in the same cluster, but even if you would:
* Only daemonSet would need a different name
* Generator configMap is unique to fake device plugin deployment, so it does not need a new name
* Rest are fine to be shared between real and fake GPU plugins

What if it would use a different namespace instead?

E.g. if one would want operator to support also fake plugin, wouldn't different namespace be easier for that, than every object having a different name?

Reply:
on that namespace topic, one idea could be to add `namespace: somefake`  into `kustomization.yaml`